book/visual-effects.md

```diff
@@ -92,7 +92,7 @@ to [WebAssembly][webassembly] to do the same.
 SDL Creates the Window
 ======================
 
-The first big task is to switch to using SDL to create the window and
+The first big task is to switch to using SDL to create windows and
```

Comment:
```suggestion
The first big task is to switch to using SDL to create the window and
```

Our browser only has one window.

Reply:
Yeah, but other browsers might have more and it keeps parallelism with “handles events”. Not critical / happy to revert if you feel strongly.

Reply:
I'd prefer to revert